templates/model-driven-operations/index.html

```diff
@@ -0,0 +1,473 @@
+      <h2 class="u-align--center u-sv4">Automate operations everywhere</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height">
+    <div class="col-4">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/04d70172-Multi-Cloud.svg",
+          alt="Multi-cloud",
+          width="263",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3>Public clouds and Multi-cloud</h3>
+      <p>Combine the advantages of public cloud services while enjoying long-term cost savings, increased flexibility and reduced vendor lock-in.</p>
+    </div>
+    <div class="col-4 ">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/9ed50294-Virtual+machine.svg",
+          alt="Virtual machine",
+          width="177",
+          height="144",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3>Bare metal and virtual machines</h3>
+      <p>Provide to your users a cloud-like experience while retaining full ownership over your data centres.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-align--center u-sv2">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/fece8a11-container-in-the-cloud.svg",
+          alt="Container",
+          width="184",
+          height="150",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <h3>Containers and orchestration</h3>
+      <p>Quickly package and deploy entire applications side-by-side with one another. Juju works seamlessly across Kubernetes, virtual machines and bare-metal servers.</p>
+    </div>
+  </div>
+</section>
+
+<hr class="is-fixed-width">
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="u-sv4 u-align--center">A vibrant ecosystem, one deploy command away</h2>
+    </div>
+  </div>
+  <div class="row u-equal-height u-sv4">
+    <div class="col-4">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/dba11aee-kubernetes.svg",
+          alt="Kubernetes",
+          width="146",
+          height="142",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+      </div>
+      <p class="p-heading--3 u-align--center">
+        <a href="/kubernetes/managed">Kubernetes</a>
+      </p>
+      <p>A curated set of tools to help you build your Kubernetes clusters from the ground up, and deliver ‘containers as a service’ across the enterprise.</p>
+    </div>
+    <div class="col-4">
+      <div class="u-hide--small u-align--center">
+        {{ 
+          image (
+          url="https://assets.ubuntu.com/v1/95abe343-image-project-openstack.png",
+          alt="Openstack",
```

Comment:
Please change the `alt` to "Charmed OpenStack with Juju is the best way to run OpenSTack"